### PR TITLE
Refactor: clean unused translations

### PR DESCRIPTION
**Pull request by ovh-ux-cds**

### CODE REVIEW

File: packages/manager/modules/cda/src/translations/Messages_es_US.json [hunk 1]
```diff
@@ -1,12 +1 @@
...
-  "cda_tab_user": "Utilisateurs",
-  "cda_tab_ip": "Control de acceso IP",
-  "cda_tab_pool": "Pools",
-  "cda_guide_title": "Guías",
-  "cda_guide_name": "Cloud Disk Array (CDA)",
-  "cda_guide_footer": "Todas las guías de OVH",
-  "cda_new_window": "Nueva ventana",
-  "cda_guide_retrieval_error": "Se ha producido un error al cargar la guía sobre Cloud Disk Array (CDA)."
-}
+{}
```
Comment (Heiiji) [inline packages/manager/modules/cda/src/translations/Messages_es_US.json:1]: why we keep a trad file with no key ? :/

Comment (marie-j) [reply packages/manager/modules/cda/src/translations/Messages_es_US.json:1]: This PR has been auto generated, maybe we should improve and delete file if empty

**Comment by antleblanc** [comment]
Hi @marie-j!

Do you think that we should merge this Pull Request?
Or maybe you are considering adding this as an automated task run as part as our CI/CD workflow?

In addition, re-run this might brings a larger diff (Pull Request opened 1+ month ago).

Thank you!

**Comment by marie-j** [comment]
> Hi @marie-j!
> 
> Do you think that we should merge this Pull Request?
> Or maybe you are considering adding this as an automated task run as part as our CI/CD workflow?
> 
> In addition, re-run this might brings a larger diff (Pull Request opened 1+ month ago).
> 
> Thank you!

@antleblanc I opened a PR to be able to review and eventually check any issues we could face ! I guess this PR could be merged and this automatically committed on `develop` :)

**Review by antleblanc — APPROVED** [review]

**Pull request merged**